packages/plugins/i18n/admin/src/translations/de.json

```
@@ -0,0 +1,64 @@
+{
+  "CMEditViewCopyLocale.copy-failure": "Beim Kopieren der Übersetzungen ist ein Fehler aufgetreten",
```

```suggestion
  "CMEditViewCopyLocale.copy-failure": "Beim Kopieren der Übersetzung ist ein Fehler aufgetreten",
```

I think this needs to be singular.